WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -163,11 +163,15 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
     }
 
     func configureNavigationButtons() {
+        addButton.accessibilityLabel = "Add new post"
```

Comment:
I think a better place for this would be the `addButton`'s initializer in here:

https://github.com/wordpress-mobile/WordPress-iOS/blob/042e124c5f82516334a27380a4a30a68f0a0f8ce/WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift#L131-L133

In this way, we can also fix the same problem in the Pages List. 

The label can also just be “Add”. There is enough context on the page that the user should be able to figure out that “Add” means adding a new post or adding a new page. You can read more about this in the [Guidelines for creating labels](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW20). 

Lastly, strings should be localized. Using `NSLocalizedString` should be enough. The translation will be handled during the release process. Please feel free to poke around in the codebase for some examples. 🙂 

- [ ] Move to the `addButton` initializer
- [ ] Use `NSLocalizedString`
- [ ] Shorten to `"Add"`